xbmc/cores/VideoPlayer/VideoRenderers/FrameBufferObject.cpp

```diff
@@ -53,10 +53,14 @@ CFrameBufferObject::CFrameBufferObject()
 
 bool CFrameBufferObject::IsSupported()
 {
+#if HAS_GLES >= 2
```

Comment:
ahh. this check is nonsense and should be removed. opengl can test for support before calling initialize() and log a reasonable message other than "error". gles does not need to test for support.